src/main/java/org/jabref/gui/copyfiles/CopyFilesAction.java

```diff
@@ -56,7 +57,7 @@ public void execute() {
                     Localization.lang("Copy linked files to folder..."),
                     Localization.lang("Copy linked files to folder..."),
                     exportTask);
-            Globals.TASK_EXECUTOR.execute(exportTask);
+            BackgroundTask.wrap(exportTask).executeWith(Globals.TASK_EXECUTOR);
```

Comment:
This doesn't seem to be equivalent to the previous code as the completion listerners of the `CopyFilesTask` are no longer invoked. Wrap `exportTask` just takes the `run` command from `exportTask`.